examples/graph_animations/demo.md

```diff
@@ -0,0 +1,61 @@
+```julia
+using Javis, LaTeXStrings
+
+function edge(; p1, p2, kwargs...)
+    # sethue("black")
+    # ob = blend(p1, p2, "orange", "blue")
+    # setblend(ob)
+    line(p1, p2, :clip)
+    rect(p1, (p2-p1)..., :fill)
+    # rect(p1, p2, :fill)
+    clipreset()
+end
+
+function node(p=O, color="black")
+    sethue(color)
+    circle(p, 10, :fill)
+    return p
+end
+
+function ground(args...) 
+    background("white")
+    sethue("black")
+end
+
+video = Video(400, 400)
+Background(1:150, ground)
+
+g = @Object 1:150 JGraph(true, 100, 100) O
```

Comment:
Seems like the `O` doesn't have any effect?
For example replacing it by `Point(-100,0)` does not shift it 100px to the left